modules/ROOT/pages/sso-use-case-1.adoc

```diff
@@ -1,12 +1,14 @@
-= Configuring SSO When No Identity Provider Is Configured in Anypoint
+= Configuring SSO When No Identity Provider Is Configured in Anypoint Platform
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This use case includes all of the instructions for setting up an identity provider for your portal when no other identity provider is configured in Anypoint Platform. You want to configure an identity provider as a login option for your portal. Since there is no identity provider configured in Anypoint Platform, the default identity provider automatically maps the identity provider for the portal to the identity provider in Anypoint Platform. This enables every Salesforce user to have an identity in Anypoint Platform.
+This use case includes instructions for setting up an identity provider for your portal when no other identity provider is configured in Anypoint Platform. You want to configure an identity provider as a login option for your portal. Since there is no identity provider configured in Anypoint Platform, the default identity provider automatically maps the identity provider for the portal to the identity provider in Anypoint Platform. This enables every Salesforce user to have an identity in Anypoint Platform. See xref:enabling-sso-for-your-portal.adoc#out-of-the-box-configurations[Out-of-the-Box Configurations] for more information about the default identity provider.
 
 == Before You Begin
-In the identity provider of your choice, the users or identities must be created for the group of the users who have access to the application. 
+* In the identity provider of your choice, the users or identities must be created for the group of the users who have access to the application. 
+
+* Review the information in the xref:enabling-sso-for-your-portal.adoc#before-you-begin [Before You Begin] section to understand the roles and permissions required and the information that API Experience Hub expects to configure custom user attributes and claims for the identity provider.
```

Comment:
```suggestion
* Review the information in the xref:enabling-sso-for-your-portal.adoc#before-you-begin [Before You Begin] section to understand the roles and permissions required and the information that API Experience Hub requires to configure custom user attributes and claims for the identity provider.
```

Reply:
In this sentence, expects is unclear to me. I read it as AEH expects to configure, not "stuff AEH expects to be able to configure" (aka requires).